ros/src/computing/perception/detection/lib/fusion/include/fusion_func.h

```diff
@@ -56,8 +54,8 @@
 #endif
 
 struct Scan_image{
-	float distance[IMAGE_WIDTH][IMAGE_HEIGHT];
-	float intensity[IMAGE_WIDTH][IMAGE_HEIGHT];
+	std::vector< std::vector <float> > distance;
+	std::vector< std::vector <float> > intensity;
 	int max_y;

```

Comment:
You need not to insert space for nested template parameter. It's C++03 style. C++11 or higher does not require spaces for it.